android/src/main/kotlin/live/hms/hmssdk_flutter/HMSErrorLogger.kt

```diff
@@ -24,6 +24,8 @@ class HMSErrorLogger {
          */
         fun returnArgumentsError(errorMessage: String): Unit? {
             HMSLogger.e("FL_HMSSDK Args Error", errorMessage)
+
+            // TODO: why is this return null required?
```

Comment:
@Decoder07 why is this return null required?

Reply:
This is required because a value needs to be returned to the value